### PR TITLE
build/configs/rtl8720e/: Release more PRSRAM to loadable apps usage, bootloader supports hotfix and fix recovery flow.

### PR DESCRIPTION
**Pull request by fengyunlai**
1.Release more PSRAM, move KR4 and KM4 text code to XIP.
    2.Bootloader support Hotfix, and fix recovery and re-write BP case flow, increase bootloader flash and ram size.
    3.Fix prepen_header.sh error when psram_2.bin is empty case.
    4.Disable BLE for now, flash operations with XIP on will have some hanging issue. This will be fixed in later PR.

### CODE REVIEW

**Comment by sunghan-chang** [comment]
@fengyunlai Thank you for the PR.
@thapav is working on this. Once she checks, let's merge.

File: build/configs/rtl8720e/scripts/rlx8720e_img2_is.ld [hunk 1]
```diff
@@ -278,7 +274,7 @@ SECTIONS
 		. = . + 4096;
 		_ebss = ABSOLUTE(.);
 
-	} > KM4_BD_PSRAM_NS_NTZ
+	} > KM4_BD_PSRAM_NS_TZ
```
Comment (thapav) [inline build/configs/rtl8720e/scripts/rlx8720e_img2_is.ld:277]: For Trustzone disabled, should this be KM4_BD_PSRAM_NS_NTZ ??

Comment (fengyunlai) [reply build/configs/rtl8720e/scripts/rlx8720e_img2_is.ld:277]: Yes you are right, let me change it back!

File: build/configs/rtl8720e/tc/defconfig [hunk 2]
```diff
@@ -260,7 +260,7 @@ CONFIG_ARCH_BOARD_HAVE_FLASH=y
 #
```
[the remark this answers is not in the document]
Comment (fengyunlai) [reply build/configs/rtl8720e/tc/defconfig:260]: Sure I updated.

**Review by sunghan-chang — APPROVED** [review]

**Pull request merged**